### ISSUE TITLE
Changing the sort order in the tooltip isn't working as expected (?)

### ISSUE THREAD

**Issue by SacredDuckwhale** [SacredDuckwhale/Rarity#177]
Source: [Curse](https://www.wowace.com/projects/rarity?comment=1861)
> Hiya! I just installed Rarity and I would like to change the sort order in the tooltip. It says use Ctrl+click to change sort order, but nothing happens when I ctrl+click. Help! Thank you :-)

and then

> So mine worked while hovering over the minimap icon itself and shift+right clicking instead of the regular shift+click (I assume left click when it says click alone), but it does say shift+click in the tooltip, so I feel ya.

Not sure what the issue is: Does it not use the correct keys? Or is the text simply wrong? Does it behave identically for LDB feeds and the mini map icon (as it should)?

Needs looking into, I guess.

**Comment by godejord** [comment]
I have given this some testing, and I am not really able to reproduce any fault here. The keys function just like they say. I suppose maybe the only issue I can think of is knowing where to click..? The texts say which combinations will change what, but it doesn't specify that you have to actually click the minimap button and not the tooltip text itself.

**Comment by SacredDuckwhale** [comment]
That was exactly what I remembered from testing myself. I remember accidentally breaking this in an alpha and then fixing it again (https://github.com/SacredDuckwhale/Rarity/commit/b9e3f1bd0e5df1d3600a654d98fccf499647d972). 

However, that doesn't match the time frame of the original comment, which was posted long before I did the refactoring (see history here: https://github.com/SacredDuckwhale/Rarity/commits/master/Utils/Sorting.lua). Therefore, it doesn't seem like this would be the problem.
